llvm/test/CodeGen/AArch64/sve-fmsub.ll

```diff
@@ -427,3 +456,6 @@ entry:
   %0 = tail call <7 x half> @llvm.fmuladd(<7 x half> %neg, <7 x half> %b, <7 x half> %neg1)
   ret <7 x half> %0
 }
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+; CHECK-SME: {{.*}}
+; CHECK-SVE: {{.*}}
```

Comment:
You should remove the CHECK-SME/SVE prefixes if they're not used (or expected to be the same).